test/dag_sampler.jl

```diff
@@ -0,0 +1,53 @@
+using Random, CausalInference, Statistics, Test, Graphs
+@testset "Dag-Zig-Zag" begin
+
+    Random.seed!(1)
+
+    N = 20 # number of data points
```

Comment:
Schau mal, hier musste ich N auch niedriger wählen als in https://github.com/mschauer/CausalInference.jl/blob/66f1d36fb15405d73c5e7cfd0175c5a9c8361cea/test/sampler.jl#L5